config.yml.erb

```diff
@@ -326,7 +326,7 @@ app_servers: {}
 rack_mini_profiler_enabled:
 https_development:
 default_scheme: '<%=(env == 'development') || ci ? 'http:' : 'https:'%>'
-allowed_iframe_ancestors: http://*.disney.com http://*.diznee.net cuantrix.mx <%=pegasus_site_host%> <%=dashboard_site_host%> curriculum.code.org codecurricula.com
+allowed_iframe_ancestors: http://*.disney.com http://*.diznee.net cuantrix.mx <%=pegasus_site_host%> <%=dashboard_site_host%> curriculum.code.org codecurricula.com https://*.lausd.iap.allhere.co
```

Comment:
- Security wise:
 I wonder if any documentation that points to the infra chat before adding any new domain. 
- Code wise I prefer a style like this (but not sure if there is a rubocop rule) since it is easier to read
```
allowed_iframe_ancestors: [
  "http://*.disney.com",
  "http://*.diznee.net",
  "cuantrix.mx",
  "<%= pegasus_site_host %>",
  "<%= dashboard_site_host %>",
  "curriculum.code.org",
  "codecurricula.com",
  "https://*.lausd.iap.allhere.co"
]
```

Reply:
+1 that makes the diffs easier to read too, and might allow inline comments?